img/app.svg

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<svg width="16" height="16" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg">
-    <path d="M13,5L12.125,14C12.064,14.549 11.552,15 11,15L5,15C4.448,15 3.936,14.549 3.875,14L3,5L13,5ZM8,6.02C6.096,6.02 4.543,7.573 4.543,9.477C4.543,11.381 6.096,12.934 8,12.934C9.904,12.934 11.457,11.381 11.457,9.477C11.457,7.573 9.904,6.02 8,6.02ZM8,6.885C9.437,6.885 10.593,8.04 10.593,9.477C10.593,10.914 9.437,12.07 8,12.07C6.563,12.07 5.407,10.914 5.407,9.477C5.407,8.04 6.563,6.885 8,6.885ZM7.809,7.287C7.665,7.29 7.549,7.408 7.549,7.552L7.181,9.471L7.181,9.476C7.102,9.803 7.326,9.909 7.576,10.061L7.579,10.062L8.874,10.744C9.156,10.961 9.482,10.538 9.2,10.321L8.432,9.481L8.432,9.477L8.083,7.555C8.083,7.404 7.96,7.283 7.809,7.287L7.809,7.287ZM9.5,1L10,2L13,2C13.554,2 14,2.446 14,3L14,4L2,4L2,3C2,2.448 2.443,2.003 3,2L6,2L6.5,1L9.5,1Z" style="fill:#fff;"/>
+<svg viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve">
```

Comment:
You mean as a standard for nextcloud or generally. Because I am pretty sure this is only, so the coordinates make sense (they are all between 0 and 16). The width I exported was 16x16.